Basket-Donation-Payments/Basket-Donation-Payments/View Controllers/Login & Signup Flow/UserSignupViewController.m

```diff
@@ -113,15 +106,37 @@ - (void)createUserToSave {
     self.user.username = self.usernameTextField.text;
     self.user.password = self.passwordTextField.text;
     self.user.email = self.emailTextField.text;
-    NSData *profilePicData = UIImageJPEGRepresentation([UIImage imageNamed:@"PlaceholderProfilePic"], 1);
-    self.user.profilePicFile = [PFFileObject fileObjectWithData:profilePicData]; //TODO: implement profile pic
-    
+    self.user.profilePicFile = [Utils getFileFromImage:self.userProfilePicImage.image];
+
     self.user.recentDonations = [NSMutableArray array];
     self.user.favoriteNonprofits = [NSMutableArray array];
     self.user.favoriteBaskets = [NSMutableArray array];
 }
 
-- (IBAction)cancelButtonTapped:(id)sender {
+#pragma mark Image Picker
+
+- (IBAction)addPictureButtonTapped:(id)sender {
+    [Utils createImagePickerVCWithVC:self];
+}
+
+- (void)imagePickerController:(UIImagePickerController *)picker didFinishPickingMediaWithInfo:(NSDictionary<NSString *,id> *)info {
+    UIImage *editedImage = info[UIImagePickerControllerEditedImage];
+    self.userProfilePicImage.image = editedImage;
+    [self dismissViewControllerAnimated:YES completion:nil];
+}
+
+#pragma mark Navigation Segues
```

Comment:
Cool trick: if you do `#pragma mark - Navigation Segues` with a hyphen you get nice separator lines in the method list in Xcode.
![Screen Shot 2020-08-05 at 2 34 15 PM](https://user-images.githubusercontent.com/1676646/89466493-cc089680-d728-11ea-9506-7eac49d2c75b.png)